### PR TITLE
create the missing container when creating blobs.

### PR DESCRIPTION
**Pull request by andyliuliming**

### CODE REVIEW

File: src/bosh_azure_cpi/lib/cloud/azure/storage/blob_manager.rb [hunk 1]
```diff
@@ -406,6 +404,17 @@ def _upload_page_blob_in_threads(file_path, file_size, container_name, blob_name
...
+      if e.inspect.include?('ContainerNotFound')
+        @blob_service_client.create_container(container_name, {})
+        @blob_service_client.create_page_blob(container_name, blob_name, file_size, options)
+      else
+        raise e
+      end
+    end
   end
 
   private
```
Comment (bingosummer) [inline src/bosh_azure_cpi/lib/cloud/azure/storage/blob_manager.rb]: I think, the `private` in line 411 should be removed and the `private` in line 299 should be kept.

Comment (andyliuliming) [reply src/bosh_azure_cpi/lib/cloud/azure/storage/blob_manager.rb]: L281 already have a private.

Comment (andyliuliming) [reply src/bosh_azure_cpi/lib/cloud/azure/storage/blob_manager.rb]: I think private in L420 is for the classes.

Comment (bingosummer) [reply src/bosh_azure_cpi/lib/cloud/azure/storage/blob_manager.rb]: yes. you are right

**Pull request merged**